model/src/main/java/org/perfrepo/dto/test_execution/TestExecutionGroupValueDto.java

```diff
@@ -0,0 +1,29 @@
+package org.perfrepo.dto.test_execution;
+
+import java.util.List;
+
+/**
+ * @author Jiri Grunwald (grunwjir@gmail.com)
+ */
+public class TestExecutionGroupValueDto {
```

Comment:
It represents a group of test execution values. This group is for each test metric, therefore the values are grouped by a metric.